docs/source/dev-guide/deep-dives/install.md

```diff
@@ -6,8 +6,6 @@ Some resources that helped draft this document:
 * https://www.anaconda.com/blog/understanding-and-improving-condas-performance
 -->
 
-(deep_dive_install)=
-
 # Deep dive: conda install
```

Comment:
Let's remove this title redundancy:

<img width="272" alt="Screen Shot 2022-11-14 at 10 25 39 AM" src="https://user-images.githubusercontent.com/4546435/201713014-22dbca71-9dd9-4dc4-bc48-01e547a83514.png">


```suggestion
# `conda install`
```